scripts/release/marketplaceRelease.js

```diff
@@ -0,0 +1,139 @@
+const nodefetch = require("node-fetch");
```

Comment:
Question: Is there a reason we do not make this part of `pluggable-widgets-tools`?

Reply:
Its not being used anywhere else besides the github action, so i dont think is really useful in the piw-tools.